debian/clearwater-etcd.init.d

```diff
@@ -127,10 +128,18 @@ join_cluster_as_proxy()
 
 setup_etcdctl_peers()
 {
-        # Build the client list based on $etcd_cluster, each entry is simply
-        # <IP>:<port> using the client port. Replace commas with whitespace,
-        # then split on whitespace (to cope with etcd_cluster values that have
-        # spaces)
+        # Build the client list based on $etcd_cluster. Each entry is simply
+        # <IP>:<port>, using the client port. Replace commas with whitespace,
+        # then split on whitespace (to cope with etcd_cluster values that have spaces)
+
+        # If we were in a working cluster before, we will have saved off an up to
```

Comment:
As discussed, I think the top comment block belongs with the second block of code